quaddtype/sdist_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -x
+
+if [ -d "build/" ]; then
+    rm -rf dist/
+fi
+
+python -m pip uninstall -y numpy_quaddtype
+python -m build --sdist --outdir dist/
+python -m pip install dist/numpy_quaddtype-0.1.0.tar.gz -v
```

Comment:
Could we avoid hard coding the version into the path?

Reply:
Reason behind that, you can't test both local build and sdist together as for changes to be visible in sdist we need to commit those changes first only then they get reflected and ready to test inside sdist. So I thought to do it separately.